blog-article.html

```diff
@@ -0,0 +1,454 @@
+                        <ul class="sidebar__list">
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Our blog 
+                                    <span class="sidebar__num">
+                                        (12)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Metaverse 
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Gaming 
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Art
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Robot 
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Lauchpad 
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Crypto 
+                                    <span class="sidebar__num">
+                                        (430)
+                                    </span>
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+                <div class="aside-box">
```

Comment:
How I see, we have different classes for the same entity. 
`aside__title` and `sidebar-title`. These two blocks looks the same and I thinking we can do the same classes here for feature

Reply:
Done